assembly/assembly-main/src/assembly/bin/launcher.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+
+# See: https://sipb.mit.edu/doc/safe-shell/
+set -e
+#set +o posix
+
+# Run the finish function if exit signal initiated
+trap exit SIGHUP SIGINT SIGTERM
+
+init_global_variables () {
+
+  # For coloring console output
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  NC='\033[0m'
+
+  ### Define various error and usage messages
+  USAGE="
+Usage:
+  launcher [COMMAND]
```

Comment:
Script has an extension. Please add extension to usage docs or remove extension from file name.